pillar/rds-articles.yaml

```diff
@@ -11,6 +11,12 @@
 
 '30274':
     date: '2030-01-01T00:00:00Z'
-    display: https://hub.stenci.la/api/projects/176/snapshots/2/article-30274.md
-    # temporarily linking to Markdown version due to lack of a generated DAR
-    download: https://hub.stenci.la/api/projects/176/snapshots/2?format=tar.gz
+    display: https://elife.stencila.io/article-30274/
+    # snapshot number missing just for launch
```

Comment:
Can you elaborate on this?

Reply:
Normally the snapshot specific URL would be used https://elife.stencila.io/article-30274/v9/. I just added this comment to clarify that this is an exception (for anyone wanting to publish future ERA articles). Do you want me to detail it more in the comment? Or remove it?

Reply:
Happy with that. I will approve this.